src/Microsoft.DncEng.SecretManager/SecretTypes/AzureDevOpsAccessToken.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using System.Globalization;
+using System.Text.RegularExpressions;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.DncEng.CommandLineLib;
+
+namespace Microsoft.DncEng.SecretManager.SecretTypes
+{
+    [Name("azure-devops-access-token")]
+    public class AzureDevOpsAccessToken : GitHubAccountInteractiveSecretType<AzureDevOpsAccessToken.Parameters>
+    {
+        private readonly TimeSpan _rotateBeforeExpiration = new TimeSpan(-15, 0, 0, 0);
+        private readonly Regex _patExpirationRegex = new Regex(@"^\s+\d{1,2}/\d{1,2}/\d{4}\s+$");
+
+        public class Parameters
+        {
+            public string Name { get; set; }
+            public string Organization { get; set; }
+            public string GitHubBotAccountSecret { get; set; }
+            public string GitHubBotAccountName { get; set; }
+        }
+
+        public AzureDevOpsAccessToken(ISystemClock clock, IConsole console) : base(clock, console)
+        {
+        }
+
+        protected override async Task<SecretData> RotateValue(Parameters parameters, RotationContext context, CancellationToken cancellationToken)
+        {
+            if (!Console.IsInteractive)
+            {
+                throw new InvalidOperationException($"User intervention required for creation or rotation of GitHub bot account.");
```

Comment:
This probably shouldn't say "GitHub bot account" when the secret isn't a github bot account.

Reply:
definitely, it's fixed now